yapapi/engine.py

```diff
@@ -841,6 +871,15 @@ async def find_offers(self) -> None:
 
         When the subscription expires, create a new one. And so on...
         """
+        # Remove some negotiable property ranges when yagna version is less than 0.10.0-rc1.
+        # This will be handled by yagna capabilities API in the future.
+        if await yagna_version_less_than("0.10.0-rc1"):
```

Comment:
I'd consider putting this logic in the MarketStrategy

Reply:
Done